### PR TITLE
Us133a borrower confirms deed correct

### PR DESCRIPTION
**Pull request by MandyNimmo**
us133a acceptance tests

### CODE REVIEW

File: features/borrower_confirms_deed.feature [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+         - Charging clause
+
+  When I navigate to the borrower frontend "/searchdeed" page
+  And I retrieve the deed id
+  And I retrieve the unique user id using the URL
+  And I search for the deed using the unique borrower token
+  And I enter the borrowers date of birth
+  Then the Your mortgage deed page is displayed
+  Then I confirm the mortgage deed
+  And a confirmation page is displayed
```
Comment (nigelpain) [inline features/borrower_confirms_deed.feature:29]: Really minor but would consider swapping the Then And on those last two lines so you've got "And I confirm the mortgage deed" and "Then a confirmation page is displayed" in my opinion it reads slightly better and also matches better with the confirm being the action and the page displayed being the result.

File: features/borrower_confirms_deed.feature [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  As a Borrower
+  I want to confirm my mortgage deed
+  So that I can complete the remortgage process
+
+Background:
+  Given I have valid deed data with <1> borrowers
+  And I create the deed via the Deed API
+
+@us133a
+Scenario: Borrower enters a valid deed reference
```
Comment (nigelpain) [inline features/borrower_confirms_deed.feature:9]: Feel the title of this scenario should reflect more the purpose of the test, this one looks a lot like the scenario name for borrower views the deed.

File: features/borrower_confirms_deed.feature [hunk 3]
```diff
@@ -0,0 +1,31 @@
+@borrower_confirms_deed
+
+Feature: Borrower confirms the Deed
+  As a Borrower
+  I want to confirm my mortgage deed
+  So that I can complete the remortgage process
+
+Background:
+  Given I have valid deed data with <1> borrowers
+  And I create the deed via the Deed API
```
Comment (nigelpain) [inline features/borrower_confirms_deed.feature]: No need for background as there is only one scenario in this feature file.

**Comment by nigelpain** [comment]
Only the three minor comments about the feature file, was going to say something about the fact that all the words on the finished screen are being checked but this is part of the acceptance criteria so fine, :+1:

File: features/borrower_confirms_deed.feature (hunk 3 above)
Comment (MandyNimmo) [inline features/borrower_confirms_deed.feature]: removed

File: features/borrower_confirms_deed.feature (hunk 2 above)
Comment (MandyNimmo) [inline features/borrower_confirms_deed.feature:9]: changed to borrower confirms deed

File: features/borrower_confirms_deed.feature (hunk 1 above)
Comment (MandyNimmo) [inline features/borrower_confirms_deed.feature:29]: done

**Comment by tsangsimon** [comment]
:+1:

**Pull request merged**